app/helpers/displayHelpers.php

```diff
@@ -838,6 +841,7 @@ function caEditorFindResultNavigation($po_request, $po_instance, $po_result_cont
 				}
 				TooltipManager::add(".next.record", "Next");
 			} else {
+				// DO somehting to load next page of results
```

Comment:
Similar to the logic for the "previous" button:

Figure out if there is a next page ($vn_total_pos > $vn_total_count?), if so display link, on click will need to load the next result page in to context? and then find the first result on that page?